src/03/SimpleAddNewForm.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component, PropTypes } from 'react';
+
+class SimpleAddNewForm extends Component {
+  constructor() {
+    super();
+  }
+
+  render() {
+    return (
+      <div className='add-new-form'>
+      </div>
+    );
+  }
+}
+
+SimpleAddNewForm.PropTypes = {
+};
+
```

Comment:
Big +1 for the default structure here :smile:.  Encouraging folks to use `propTypes` in particular is great.